presto-main/src/main/java/com/facebook/presto/type/DecimalOperators.java

```diff
@@ -37,6 +37,7 @@
 import static com.facebook.presto.metadata.FunctionKind.SCALAR;
```

Comment:
For the numbers in the commit message, can you show the delta before and after the change? Absolute numbers don't mean much, since they are dependent on a number of external factors.